src/configure.ac

```diff
@@ -354,7 +399,7 @@ dnl AC_CHECK_SIZEOF([int *function()]
 # pa-risc.h
 # ppc32.h
 # ppc32alt.h (-maltivec)
-# ppc64.h (-m64)
+# ppc64.h (-m64 -maltivec)
 # ppc64alt.h (-maltivec -faltivec)
```

Comment:
Weird. Why are we listing `-maltivec` along with `ppc64.h` (not only `ppc64alt.h`), but also list `-faltivec` only along with the other? I guess for no specific reason?

Reply:
In jumbo land, ppc64.h was planned on being the only header needed.  but if there are 3 options (normal, Altivec1, Altivec2), then I may not be able to pack it all in one header.

Reply:
It should be possible to pack all 3 in one header.